projects/media-viewer/src/lib/toolbar/media-viewer-toolbar/media-viewer-toolbar.component.html

```diff
@@ -0,0 +1,193 @@
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvPresentBtn.id)" aria-pressed="false"
+          (click)="togglePresentBar()"><span data-l10n-id="icpMode_label">Present</span>
+        </button>
+
+        <button *ngIf="enableRedaction && toolbarButtons.showRedact" [disabled]="icpEnabled" id="mvRedactBtn"
+          #mvRedactBtn title="Redact" data-l10n-id="redact" class="mv-button mv-toolbar__menu-button--redact"
+          [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvRedactBtn.id)"
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvRedactBtn.id)" aria-pressed="false"
+          (click)="toggleRedactBar()">
+          <span>Redact</span>
+        </button>
+
+        <button *ngIf="toolbarButtons.showGrabNDragButton" [disabled]="icpEnabled" id="mvGrabBtn" #mvGrabBtn
+          class="mv-button mv-toolbar__menu-button--grab"
+          [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvGrabBtn.id)"
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvGrabBtn.id)" aria-pressed="false"
+          (click)="toggleGrabNDrag()">
+          <span>Grab and drag</span>
+        </button>
+
+        <button *ngIf="toolbarButtons.showDownload" [disabled]="icpEnabled || redactionEnabled" id="mvDownloadBtn"
+          #mvDownloadBtn class="mv-button mv-toolbar__menu-button--download" title="Download" data-l10n-id="download"
+          [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvDownloadBtn.id)"
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvDownloadBtn.id)" aria-pressed="false"
+          (click)="downloadFile()">
+          <span>Download</span>
+        </button>
+
+        <button *ngIf="toolbarButtons.showPrint" [disabled]="icpEnabled || redactionEnabled" id="mvPrintBtn" #mvPrintBtn
+          title="Print" data-l10n-id="print" class="mv-button mv-toolbar__menu-button--print"
+          [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvPrintBtn.id)"
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvPrintBtn.id)" aria-pressed="false"
+          (click)="printFile()">
+          <span>Print</span>
+        </button>
+
+        <button *ngIf="enableAnnotations && toolbarButtons.showCommentSummary" [disabled]="redactionEnabled"
+          id="mvCommentsBtn" #mvCommentsBtn class="mv-button mv-toolbar__menu-button--comments" title="Comments"
+          data-l10n-id="comments" [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvCommentsBtn.id)"
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvCommentsBtn.id)" aria-pressed="false"
+          (click)="toggleCommentsPanel()">
+          <span>Comments</span>
+        </button>
+
+        <!-- TODO: Where is this and how does it work?
+      <ng-container *ngIf="!icpEnabled">
+        <a *ngIf="toolbarButtons.showBookmark"
```

Comment:
what is this?

Reply:
As discussed, this is legacy code and will be removed in the next commit.